src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -1016,6 +1016,43 @@ class IpexCommunicationService extends AgentService {
       };
     }
   }
+
+  async getLinkedGroupFromIpexApply(id: string, credentialSaid: string) {
```

Comment:
Can we instead just have `(id: string)` and this function return all credentials?

So it can like:
```
{
 [credSaid1]: { accepted: false, membersJoined: [...] },
 [credSaid2]: { accepted: false, membersJoined: [...] }
}
```